src/main/kotlin/no/nav/syfo/domain/model/UnpublishedVarsel.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.syfo.domain.model
+
+import no.nav.syfo.domain.PersonIdent
+import java.util.UUID
+
+data class UnpublishedVarsel(
+    val personident: PersonIdent,
+    val varselUuid: UUID,
+    val journalpostId: String,
+)
```

Comment:
Er det denne vi kalte `ExpiredVarsel` i `isaktivitetskrav`? Det eneste jeg tenker er litt rart med `Unpulished` er at det er en tilstand som vi endrer. Hva er/hvor lever dataklassen med en gang den blir published? `Expired` var jo også en tilstand, men det var aldri noe man kunne "flippe" tilbake til å være "unexpired" på en måte 🤔 Et expired varsel vil for alltid ha passert fristen den fikk.

Reply:
Nei, `ExpiredVarsel` var vel for varsler med utløpt frist. Dette er kun for å holde på det vi trenger å hente ut fra et journalført `Varsel` slik at vi kan sende hendelse (med personident, uuid og journalpostId)) til esyfovarsel. Kunne sikkert bare brukt en variant av Pair<PersonIdent, Varsel> her også, men da sier ikke typen noe om at journalpostId må være satt.

Reply:
Aha, ja selvfølgelig! Det var jeg som misforstod. Disse går jo ut "med en gang".

Reply:
Også litt enig i at det kan være forvirrende med `UnpublishedVarsel`. Da kan vi ender vi litt opp med to domenemodeller som kan gjenspeile samme entitet 🤔

Reply:
Jepp, kan refaktorere til å bruke Varsel-domeneobjektet når https://github.com/navikt/isarbeidsuforhet/pull/20 er merget

Reply:
Refaktorert nå @eirikdahlen @vetlesolgaard

Reply:
Nydelig! 😍